js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,17 @@
+function alternatingCharacters(arr) {
+  let result=[]
+  arr.map((value)=>{
+    let string=value.split("");
+    let past=string[0];
```

Comment:
Instead of `past`  better name would be `previousCharacter` with the context of this question.